torch/csrc/jit/runtime/logging.h

```diff
@@ -61,7 +61,7 @@ struct JITTimePoint {
 };
 
 TORCH_API JITTimePoint timePoint();
-TORCH_API void recordDurationSince(const std::string& name, JITTimePoint tp);
+TORCH_API void recordDurationSince(const std::string& name, const JITTimePoint& tp);
```

Comment:
clang-format thinks that this line is too long.
```suggestion
TORCH_API void recordDurationSince(const std::string& name,
                                   const JITTimePoint& tp);
```